src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background: #fff;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
+  order: 1;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  height: 100%;
+}
+
+.nav__item,
```

Comment:
why do you duplicate styles for nav__item and nav__link?

Reply:
Soo, I have tried to use just one of them and in the result, it was going something wrong with the fonts

Reply:
Remove redundant duplicate of codes. If you don't use something it should be deleted